### ISSUE TITLE
Issue about Me Pattern Buffer

### ISSUE THREAD

**Issue by dustiit** [GregTechCEu/GregTechCEu-1.19#2084]
### Checked for existing issues

- [X] I have checked for existing issues, and have found none.

### Tested latest version

- [X] I have checked that this occurs on the latest version.

### GregTech CEu Version

1.4.4

### Minecraft Version

1.20.1

### Recipe Viewer Installed

EMI

### Environment

Singleplayer

### Cross-Mod Interaction

No

### Other Installed Mods

GTM things, AE2, EMI and other mods GTM needed

### Expected Behavior

Pattern should show in AE terminal when every time enter the game__

### Actual Behavior

Pattern in ME patter hatch ( also the proxy I guess?) do not show in AE terminal when every time enter the game

### Steps to Reproduce

I tried to put the encoded pattern in me pattern hatch and t is shown normaly in AE terminal but  since I open the game again the pattern cant be seen in terminal

### Additional Information

no

**Comment by dustiit** [comment]
btw, if u take out the pattern and put it into hatch agian the hatch cant work (the multis wouldn't process the recipe)

**Comment by Drigues1901** [comment]
I am also having similar issues with the same block on ATM9, seems to me that it acts as a normal interface and just sends the items for multiple crafting recipes, and the MB gets confused. I think having a sort of blocking mode or some fix where it can send the items for the craft and diferenciate between recipes should fix it

**Comment by screret** [comment]
yes, we're in the process of making it be distinct per pattern but that isn't done yet.
